.github/CODEOWNERS

```diff
@@ -26,8 +26,8 @@
 /src/platform_impl/web              @daxpedda
 
 # Windows
-/src/platform/windows.rs            @msiglreith
-/src/platform_impl/windows          @msiglreith
+# /src/platform/windows.rs            
+# /src/platform_impl/windows          
```

Comment:
Put me on as Windows maintainer for now.

Reply:
done, thank you!